client/rest-high-level/src/main/java/org/elasticsearch/client/ml/GetDataFrameAnalyticsRequest.java

```diff
@@ -50,6 +54,14 @@ public GetDataFrameAnalyticsRequest(String... ids) {
         return ids;
     }
 
+    public Boolean getAllowNoMatch() {
+        return allowNoMatch;
+    }
+
+    public void setAllowNoMatch(Boolean allowNoMatch) {
```

Comment:
Could this "allowNoMatch" parameter be just "boolean"?
My thinking is that if the user calls this method they should decided on "true" or "false" and should not have a need to put "null".

Reply:
Good point!